packages/template-retail-react-app/app/hooks/use-add-to-cart-modal.js

```diff
@@ -172,7 +172,10 @@ export const AddToCartModal = () => {
                                             <DisplayPrice
                                                 discountPriceProps={{as: 'p'}}
                                                 basePrice={lineItemBasePrice * quantity}
-                                                discountPrice={lineItemDiscountPrice * quantity}
+                                                discountPrice={
+                                                    typeof lineItemDiscountPrice === 'number' &&
+                                                    lineItemDiscountPrice * quantity
+                                                }
```

Comment:
`discountPrice` evaluate to `false` when `lineItemDiscountPrice` is not a number.

Reply:
The [DisplayPrice component](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/1e22e16cd0a6d622f1121c63e78bd0ae8f49d7f3/packages/template-retail-react-app/app/components/display-price/index.jsx) always checks `typeof displayPrice === 'number'`, so this is technically fine, although it would be more conventional to use `null` or `undefined` for "unset".